db/seeds.rb

```diff
@@ -49,13 +49,13 @@
 Planhabit.create(plan_id: plan2.id, habit_id: habit16.id)
 
 
-plan3 = Plan.create(plan_name: "Launch Prep-Elite Edition", author: "Paul M. Thiebauth", author_info: "Web Developer - Boston, MA")
+plan3 = Plan.create(plan_name: 'Launch Prep-Elite Edition', author: 'Paul M. Thiebauth', author_info: 'Web Developer - Boston, MA')
```

Comment:
Line is too long. [131/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.